### PR TITLE
Passing selected range with codeactions

### PR DESCRIPTION
**Pull request by mat-mcloughlin**

### CODE REVIEW

File: lib/omnisharp-atom/features/code-action.ts [hunk 1]
```diff
@@ -84,6 +85,25 @@ class CodeAction implements OmniSharp.IFeature {
         }));
     }
 
+    private getRequest(client: OmniSharp.ExtendApi, codeAction?: number) {
+        var editor = atom.workspace.getActiveTextEditor();
+        var range = editor.getSelectedBufferRange();
+
+        var request = client.makeDataRequest<OmniSharp.Models.CodeActionRequest>({
+            WantsTextChanges: true,
+            SelectionStartLine: range.start.row + 1,
```
Comment (nosami) [inline lib/omnisharp-atom/features/code-action.ts]: Are these +1's still valid? I know @david-driscoll recently did some work on node-client to make indexes 0 based

Comment (mat-mcloughlin) [inline lib/omnisharp-atom/features/code-action.ts]: It didn't work without them. Think he just did the Line and Column not the selection stuff

Comment (david-driscoll) [inline lib/omnisharp-atom/features/code-action.ts]: It should apply to SelectionStart/End as well.

Have you rebased against master in a while?

Comment (david-driscoll) [inline lib/omnisharp-atom/features/code-action.ts]: Then I look at the branch and it's been rebased... very odd I'll have to take it down locally and test.

Comment (nosami) [inline lib/omnisharp-atom/features/code-action.ts]: The PR is wrong... I just tested it.

Comment (mat-mcloughlin) [inline lib/omnisharp-atom/features/code-action.ts]: hmm, i'll double check but it seems to be working for me

Comment (mat-mcloughlin) [inline lib/omnisharp-atom/features/code-action.ts]: Checked, Looked at the request that comes in and unless I add the `+1` it comes in zero based.

So I'm a little confused

**Comment by nosami** [comment]
Did you update the deps? Specifically omnisharp-client.

**Comment by mat-mcloughlin** [comment]
lol, good point

**Comment by nosami** [comment]
nice. cheers!

**Pull request merged**